x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/components/empty_state.test.tsx

```diff
@@ -21,7 +23,7 @@ describe('EmptyState', () => {
 
     expect(wrapper.find('h2').text()).toEqual('Add your first documents');
     expect(wrapper.find(EuiButton).prop('href')).toEqual(
-      expect.stringContaining('/indexing-documents-guide.html')
+      expect.stringContaining(`${docLinks.appSearchIndexingDocs}`)
```

Comment:
```suggestion
      expect.stringContaining(docLinks.appSearchIndexingDocs)
```
No need for a template literal here